src/pages/resultsView/plots/PlotsTabUtils.tsx

```diff
@@ -799,7 +799,7 @@ export function makeAxisDataPromise(
                     molecularProfileIdToMolecularProfile);
             }
             break;
-        case AlterationTypeConstants.TREATMENT_RESPONSE:
+        case AlterationTypeConstants.GENERIC_ASSAY:
             if (selection.treatmentId !== undefined && selection.dataSourceId !== undefined) {
                 ret = makeAxisDataPromise_Treatment(
                     selection.treatmentId, selection.dataSourceId, treatmentMolecularDataCachePromise,
```

Comment:
Here GENERIC_ASSAY is linked to `selection.treatmentId`. This combination of generic_assay linked to treatment should no longer occur.